javascript/src/seedsource/containers/ShapefileConstraint.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux'
+import ShapefileConstraint from 'seedsource/components/ShapefileConstraint'
+import { removeConstraint } from '../../actions/constraints'
+
+const mapStateToProps = () => {
```

Comment:
I think you can leave this out completely if you don't need to map props, and just call:

```javascript
export default connect(null, mapDispatchToProps)(ShapefileConstraint)
```